docs/reference/mapping/types/token-count.asciidoc

```diff
@@ -50,9 +50,10 @@ GET my_index/_search
 
 [NOTE]
 ===================================================================
-Technically the `token_count` type sums position increments rather than
+Technically, the `token_count` type sums position increments rather than
 counting tokens. This means that even if the analyzer filters out stop
-words they are included in the count.
+words they are included in the count.  
+If you don't want to count stop words, set option `enable_position_increments` to `false`
```

Comment:
this is redundant with the definition below and it's a little bit confusing since it impacts not only the stop filter ?